e2e/tests/ssm.test.js

```diff
@@ -216,5 +217,95 @@ describe('ssm', async () => {
       expect(result).to.not.equal(undefined)
       expect(result.body.status.status).to.contain('namespace does not allow to assume role let-me-be-root')
     })
+    it('should not pull from ssm after waiting POLLER_INTERVAL_MILLISECONDS', async () => {
+      backends.systemManager.checkUpdated = true
```

Comment:
The controller runs in a different pod/process than the e2e test. You can't change the behavior of it like this.